#vector motions
float32 v_x
float32 v_y
float32 v_pitch
#roll grasp values of EE
float32 roll
float32 grasp
#last pose of EE
float32 lpos_x
float32 lpos_y
float32 lpos_pitch